registration/register.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
+  <link rel="stylesheet" type="text/css" href="frontend/main.css">
+  <title>Registration For SPECTRA</title>
+</head>
+<body>
+  <iframe id="typeform-full" width="100%" height="100%" frameborder="0" src="https://ingochris.typeform.com/to/gltwlZ"></iframe>
```

Comment:
Please sync with marketing team about using airtable vs. typeform.. I want everyone on the same page about this before we merge your PR.